test/test_ea_mu_plus_lambda.py

```diff
@@ -221,3 +221,46 @@ def test_create_new_parent_population(population_params, genome_params, ea_param
     # we picked the first three individuals
     new_parents = ea._create_new_parent_population(3, pop.parents)
     assert new_parents == pop.parents[:3]
+
+
+def test_update_n_objective_calls(population_params, genome_params, ea_params):
+    def objective(individual):
+        individual.fitness = float(individual.idx)
+        return individual
+
+    # create a population and an instance of the evolutionary algorithm and
+    # check that number of objective calls is initially zero
+    pop = cgp.Population(**population_params, genome_params=genome_params)
+    ea = cgp.ea.MuPlusLambda(**ea_params)
+    assert ea.n_objective_calls == 0
```

Comment:
```suggestion
    # check that number of objective calls is initially zero
    n_objective_calls_expected = 0
    pop = cgp.Population(**population_params, genome_params=genome_params)
    ea = cgp.ea.MuPlusLambda(**ea_params)
    assert ea.n_objective_calls == n_objective_calls_expected
```
i would introduce this `_expected` variable and set it always according to what you expect such that you can use it in the `assert`s. i think this would make the code clearer and also the comments obsolete.